certdb/sql/database_accessor.go

```diff
@@ -220,56 +204,37 @@ func (d *Accessor) InsertOCSP(rr certdb.OCSPRecord) error {
 		return wrapSQLError(fmt.Errorf("%d rows are affected, should be 1 row", numRowsAffected))
 	}
 
-	return nil
+	return err
 }
 
 // GetOCSP retrieves a certdb.OCSPRecord from db by serial.
-func (d *Accessor) GetOCSP(serial, aki string) (rrs []certdb.OCSPRecord, err error) {
+func (d *Accessor) GetOCSP(serial, aki string) (ors []certdb.OCSPRecord, err error) {
 	err = d.checkDB()
 	if err != nil {
 		return nil, err
 	}
 
-	rr := certdb.OCSPRecord{}
-	rows, err := d.db.Query(fmt.Sprintf(selectOCSPSQL, sqlstruct.Columns(rr)), serial, aki)
+	err = d.db.Select(&ors, fmt.Sprintf(d.db.Rebind(selectOCSPSQL), sqlstruct.Columns(certdb.OCSPRecord{})), serial, aki)
 	if err != nil {
 		return nil, wrapSQLError(err)
 	}
-	defer rows.Close()
-
-	if rows.Next() {
-		err = sqlstruct.Scan(&rr, rows)
-		if err != nil {
-			return nil, wrapSQLError(err)
-		}
-		rrs = append(rrs, rr)
-	}
-	return rrs, nil
+
+	return ors, nil
 }
 
 // GetUnexpiredOCSPs retrieves all unexpired certdb.OCSPRecord from db.
-func (d *Accessor) GetUnexpiredOCSPs() (rrs []certdb.OCSPRecord, err error) {
+func (d *Accessor) GetUnexpiredOCSPs() (ors []certdb.OCSPRecord, err error) {
```

Comment:
thanks. my initial intention was name it R(esponse)R(ecord)s, but never correct it after I rename the table.